assets/src/edit-story/karma/fixture/containers/common/color.js

```diff
@@ -28,15 +28,15 @@ export class Color extends Container {
   }
 
   get button() {
-    return this.getByRole('button', { name: /Edit: /i });
+    return this.getByRole('button', { name: /Color/i });
```

Comment:
Do these need to be regexes or can they be strings now?

Reply:
Yup, updated and tested.